src/getQueryParam.ts

```diff
@@ -0,0 +1,16 @@
+export const getQueryParam = (param: string): string | number | null => {
+    if (window.location && window.location.href) {
```

Comment:
With this function, you will introduce a dependency of browser environment.

Reply:
Even if I check if window exists?